mixer/doc/adapters.md

```diff
@@ -84,18 +84,34 @@ Every template also has two additional properties associated with it:
 wants to consume `Instance` objects associated with a particular template. The template name is also used within
 operator config to provide template-specific fields to attribute mapping, which is used to create `Instance `objects.
 
-* **Template Variety**: Every template has a specific `template_variety` which can be either Check, Report or Quota.
-The template and its variety determine the signatures of the methods that adapters must implement for consuming the
-associated instances. The `template_variety` also determines under which of the core Mixer behaviors, check report or
-quota, the `instances` for the templates should be created and dispatched to adapters.
+* **Template Variety**: Every template has a specific `template_variety` which can be either Check, Report, Quota or
+AttributeGenerator. The template and its variety determine the signatures of the methods that adapters must implement
+for consuming the associated instances. The `template_variety` also determines under which of the core Mixer behaviors,
+the `instances` for the templates should be created and dispatched to adapters.
+For example:
+    * *Check* template variety instances are created and dispatched only during Mixer's Check API call.
+    * *Report* template variety instances are created and dispatched only during Mixer's Report API call.
+    * *Quota* template variety instances are created and dispatched only during Mixer's Quota API call.
```

Comment:
There is no QUota API. Quota is part of the CHeck API

Reply:
D